Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
It’s odd. The compiler knows about #if os(Android) giving an error if you mistype it but not @available(Android, unavailable)